vars/cloudfront.var.yml

```diff
@@ -11,9 +11,9 @@ cloudformation_stacks:
     template_jinja: "{{ template }}.j2"
     template_parameters:
       OriginReadTimeout: 180 #service increase requested, we now have 3 min wait available
-      WebOrigin: "{{ Environment }}{{ service_name }}WebElbDNSName"
+      WebOrigin: "{{ service_name_lower }}.{{ RootDomain }}"
       CmsOrigin: "{{ CmsOrigin }}"
-      WebACLId: "{{ Environment }}CKANWebACLId"
+      WebACLId: "{{ lookup('aws_ssm', '/config/CKAN/' + Environment + '/common/waf_acl_id', region='us-east-1') }}"
```

Comment:
There is a way to make this lookup on cfn template. But you do need to give different param string per env for it to work. See repo site-myqldgovau iam.cfn.yml file in the .GitHub/aws folder

Reply:
For now I would probably choose to keep the style consistent with other templates in this repo.